### DISCUSSION TITLE
Clarification of Don’t combine ScaledObject with Horizontal Pod Autoscaler (HPA)

### DISCUSSION THREAD

**Discussion by carmensarlo** [kedacore/keda#5108] [Q&A / Need Help]
I am seeking clarification of this statement in the FAQ. 
Does that mean migrate all deployments that are using HPA to scale on CPU or memory to KEDA ScaledObject?

Can I have deployment-A using ScaledObject with CPU trigger and deployment-B not using ScaledObject but using HPA to scale on CPU utilization? All deployments are in the same cluster.

Also can I have deployment-C use ScaledObject with CPU and another trigger, other deployments still using HPA (no ScaledObject)?

**Comment by ayushk048-Tandem** [comment]
Having the same issue with using HPA and Keda for services [#5220](https://github.com/kedacore/keda/discussions/5220)

**Reply by JorTurFer** [reply]
> Can I have deployment-A using ScaledObject with CPU trigger and deployment-B not using ScaledObject but using HPA to scale on CPU utilization? All deployments are in the same cluster.

Yes, each workload is independent. You cannot have both (ScaledObject and HPA) scaling same workload (neither 2 HPA or 2 ScaledObjects) but you can scale each workload with one on them as you wish. Deployment A with ScaledObject, deployment B with HPA, etc...

> Also can I have deployment-C use ScaledObject with CPU and another trigger, other deployments still using HPA (no ScaledObject)?

Yeah, each workload is independent from the others
